mllib/src/main/scala/org/apache/spark/ml/r/AFTSurvivalRegressionWrapper.scala

```diff
@@ -30,6 +32,7 @@ import org.apache.spark.ml.regression.{AFTSurvivalRegression, AFTSurvivalRegress
 import org.apache.spark.ml.util._
 import org.apache.spark.sql.{DataFrame, Dataset}
 
+@Since("2.0.0")
 private[r] class AFTSurvivalRegressionWrapper private (
```

Comment:
Yes we can, but it takes some time... Finding the API is done by the tool, but adding the annotation was done manually.